packages/experiments/src/components/Button/Button.styles.ts

```diff
@@ -321,6 +329,12 @@ export const ButtonStyles: IButtonComponent['styles'] = (props, theme, tokens):
           },
           [`:hover:active .${globalClassNames.icon}`]: {
             color: tokens.iconColorPressed
+          },
+          [`.${IsFocusVisibleClassName} &:focus`]: {
```

Comment:
Can you use `getFocusStyle` here instead?

Reply:
Per our offline discussion `getFocusStyle` cannot be used for this scenario. Updated PR with a comment on why this happens.